Dockerfile

```diff
@@ -4,19 +4,20 @@ ADD .git ./.git
 RUN git describe --abbrev=0 --tags | tee ./version
 
 
-FROM node:lts-alpine AS builder-web
-ADD gui /build/gui
-WORKDIR /build/gui
-RUN echo "network-timeout 600000" >> .yarnrc
+# FROM node:lts-alpine AS builder-web
```

Comment:
After this changed, this Dockerfile is not usable for those who want build docker images by themselves

Reply:
According https://github.com/docker/setup-qemu-action/issues/110
Git hub action will always timeout for qemu armv7 which is unnecessary.
And yes. Those who want build docker image they need build frontend first and then do docker build.
I will create a docker dev file to support local build.

Reply:
I am afraid I can't agree with you. 
>It cause build always failed in git action https://github.com/v2rayA/v2rayA/actions/runs/6764494537/job/18563292110

In this log, I see the build was cancelled by something, and the GUI built succeed.
![image](https://github.com/v2rayA/v2rayA/assets/19235246/b493cc8e-e9f7-4092-bf3a-3690a19026ed)

Reply:
<img width="493" alt="image" src="https://github.com/v2rayA/v2rayA/assets/20839187/8e74d07f-e325-41e4-a403-2b2faa422c3a">
<img width="466" alt="image" src="https://github.com/v2rayA/v2rayA/assets/20839187/644bb42a-039f-4732-8b2b-c3fb78b93f55">
Build failed due to timeout. It takes 6 hours on arm/v7 platform doing yarn and yarn build.